rasterio/_crs.pyx

```diff
@@ -136,23 +247,28 @@ class _CRS(UserDict):
         Returns
         -------
         CRS
+
         """
         if int(code) <= 0:
             raise CRSError("EPSG codes are positive integers")
-        return cls(init="epsg:%s" % code, no_defs=True)
+        return cls.from_dict(init="epsg:{}".format(code))
```

Comment:
Based off of changes in the PROJ.4 library, I would actually recommend not using the PROJ.4 syntax and instead use the WKT syntax here and use the `from_wkt` function.

Reply:
@snowman2 but what is the WKT syntax for "pull record xxxx from the epsg table?"? I think from_dict will be fine for a few versions of GDAL, there's no plan to remove OSRImportFromPROJ4 from the API. That said, I could simplify things by using from_proj4 here.

Reply:
I was thinking about the format that is not necessarily WKT, but works with the FromUserInput (i.e. `epsg:4326`). The reasoning behind my suggestion is that they behave slightly different in `PROJ.4`. And if PROJ.4 will be the engine behind GDAL, it may be slightly different there as well.

The difference is in the axis information:
```
>>> from pyproj import CRS
>>> CRS(init='epsg:4326')
<CRS: +init=epsg:4326>
Name: WGS 84
Ellipsoid:
- semi_major_metre: 6378137.00
- semi_minor_metre: 6356752.31
- inverse_flattening: 298.26
Area of Use:
- name: World
- bounds: (-180.0, -90.0, 180.0, 90.0)
Prime Meridian:
- longitude: 0.0000
- unit_name: degree
- unit_conversion_factor: 0.01745329
Axis Info:
- Longitude[lon] (east) EPSG:9101 (radian)
- Latitude[lat] (north) EPSG:9101 (radian)

>>> CRS('epsg:4326')
<CRS: epsg:4326>
Name: WGS 84
Ellipsoid:
- semi_major_metre: 6378137.00
- semi_minor_metre: 6356752.31
- inverse_flattening: 298.26
Area of Use:
- name: World
- bounds: (-180.0, -90.0, 180.0, 90.0)
Prime Meridian:
- longitude: 0.0000
- unit_name: degree
- unit_conversion_factor: 0.01745329
Axis Info:
- Geodetic latitude[Lat] (north) EPSG:9122 (degree)
- Geodetic longitude[Lon] (east) EPSG:9122 (degree)
```
The one without `init` in it behaves how the current version of GDAL behaves. However, I am not sure if that will continue to be the case in future versions.